open-api/rest-catalog-open-api.yaml

```diff
@@ -2615,6 +2888,23 @@ components:
           additionalProperties:
             type: string
 
+    CreateScanRequest:
+      type: object
+      required:
+        - select
+        - options
+      properties:
```

Comment:
This seems to be missing a few important ones, like snapshot ID, timestamp, etc.